### ISSUE TITLE
ECmean4 not running with python 3.11

### ISSUE THREAD

**Issue by oloapinivad** [oloapinivad/ECmean4#55]
So far it is not possible to run with py311 since there is a conflict between xESMF, python and esmpy. 
xESMF can't work with the most recent version of ESMPY, so that we run with 8.3.1. However, 8.3.1 does not run with python 3.11

This should be addressed in days by xESMF https://github.com/pangeo-data/xESMF/issues/218

**Comment by oloapinivad** [comment]
The conflict handling by conda is terrible.
I tested it with mamba and it work much better. A recent version of `xesmf` conflicts with `numba`, which has become a new dependency - although I am not sure where this comes from. 

```
(ECmean4) paolo@wilma:~/ECmean4$ mamba env create -n prova -f mamba_environment.yml
conda-forge/noarch                                  10.7MB @   4.2MB/s  2.9s
conda-forge/linux-64                                28.7MB @   4.0MB/s  8.1s


Looking for: ['python==3.11', 'numpy', 'netcdf4', 'dask', 'xarray', 'xesmf=0.7.0', 'esmpy', "matplotlib-base[version='>=3.5.0']", 'metpy', 'tabulate', 'cfgrib', 'seaborn', 'numba', 'pytest']


Encountered problems while solving:
  - package xesmf-0.7.0-pyhd8ed1ab_0 requires numba >=0.55.2, but none of the providers can be installed
```

BTW, we should and recommend to use mamba instead of conda, performances and error handling is way better.

**Comment by oloapinivad** [comment]
This is still an open issue, the underlying issue is that there is no working version for `numba` in python 3.11. This is required by `sparse` (indirectly by `xesmf` I think).

**Comment by oloapinivad** [comment]
Currently addressed in #70, we can do it work in local but not on the actions - probalby some conflicts in channels
